modules/cnf-topology-aware-lifecycle-operator-precache-feature.adoc

```diff
@@ -0,0 +1,270 @@
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:      4.9.18
+Digest:    sha256:1234abcd5678efgh
+Created:   2022-01-26T14:50:25Z
+OS/Arch:   linux/amd64
+Manifests: 522
+
+Pull From: example.io/openshift-release/ocp-release@sha256:1234abcd5678efgh
+----
+
+. Create the `PolicyGenTemplate` CR in `inform` mode, then save the YAML in the `du-upgrade.yaml` file.
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "du-upgrade"
+  namespace: "ztp-group-du-sno"
+spec:
+  bindingRules:
+    group-du-sno: ""
+  mcp: "master"
+  remediationAction: inform
+  sourceFiles:
+    - fileName: ClusterVersion.yaml
+      policyName: "platform-upgrade"
+      metadata:
+        name: version
+      spec:
+        channel: "candidate-4.9"
+        desiredUpdate:
+          version: "4.9.18"
+          image: "example.io/openshift-release/ocp-release@sha256:1234abcd5678efgh" <1>
+----
+<1> The `image` field contains a container image in digest format to facilitate the authentication of the image.
+
+.Verification
+
+. Check if the policies were created by running the following command:
++
+[source,terminal]
+----
+$ oc get policies -A | grep platform-upgrade
+----
++
+.Example output
++
+[source,terminal]
+----
+cnfdb1                ztp-group-du-sno.du-upgrade-platform-upgrade        inform               NonCompliant       3d18h
+cnfdb2                ztp-group-du-sno.du-upgrade-platform-upgrade        inform               NonCompliant       3d18h
+ztp-group-du-sno      du-upgrade-platform-upgrade                         inform               NonCompliant       3d18h
+----
+
+. Check if the cluster contains the desired cluster version by running the following command:
++
+[source,terminal]
+----
+$ oc get policy ztp-group-du-sno.du-upgrade-platform-upgrade -n cnfdb1 -o jsonpath='{.spec}' | jq
+----
++
+.Example output
++
+[source,terminal]
+----
+"spec": {
+                  "channel": "candidate-4.9",
+                  "desiredUpdate": {
+                    "force": false,
+                    "image": "example.io/openshift-release/ocp-release@sha256:1234abcd5678efgh",
+                    "version": "4.9.18" <1>
+                  }
+----
+<1> The `version` field shows the desired cluster version.
+
+[id="talo-precache-start_and_update_{context}"]
+== Starting the pre-caching task
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Create a policy with pre-caching enabled
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR with the `preCaching` field set to `true`, and then save the YAML in the `clustergroupupgrades-group-du.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4918
+  namespace: ztp-group-du-sno
+spec:
+  preCaching: true <1>
+  deleteObjectsOnCompletion: true
```

Comment:
I think deleteObjectsOnCompletion is deprecated (see https://bugzilla.redhat.com/show_bug.cgi?id=2040874). Now it should be as below (under spec):

```
spec:
  actions:
    afterCompletion:
      deleteObjects: true
```

Reply:
Updated all instances